client.go

```diff
@@ -221,35 +222,28 @@ func (r *marathonClient) apiDelete(uri string, post, result interface{}) error {
 }
 
 func (r *marathonClient) apiCall(method, uri string, body, result interface{}) error {
+	var jsonBody []byte
```

Comment:
oh ... i always preferred the C approach and chunk everything at the top :-) ... I'll stick to the default though!